.storybook/main.ts

```diff
@@ -71,20 +73,17 @@ const webpackConfig = (config) => {
 }
 
 module.exports = {
-  core: {
-    builder: 'webpack5',
-  },
   stories: ['../src/**/*.stories.@(ts|tsx)'],
   addons: ['@storybook/addon-essentials', '@storybook/addon-a11y'],
-  typescript: {
-    check: false,
-    checkOptions: {},
-    reactDocgen: 'react-docgen-typescript',
-    reactDocgenTypescriptOptions: {
-      tsconfigPath: path.resolve(__dirname, '../tsconfig.json'),
-    },
-  },
   webpackFinal: async (config) => {
     return webpackConfig(config)
   },
-}
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: { strictMode: false },
+  },
+  docs: {
+    autodocs: true,
```

Comment:
Suggestion: I don't think we want docs generated for every component, could you update this to be `autodocs: 'tag',` instead.

```suggestion
    autodocs: 'tag',
```